.github/workflows/ci.yml

```diff
@@ -3,7 +3,7 @@ name: ⚙️ CI
 on:
   push:
     branches:
-      - "main"
+      # - "main"
```

Comment:
```suggestion
      - "main"
```

Reply:
That was [temporary](https://github.com/getsentry/sentry-godot/pull/37#issuecomment-2520008953)
Dropped.